### DISCUSSION TITLE
Github push seems broken today on all my projects

### DISCUSSION THREAD

**Discussion by keesh0** [community/community#43234] [Projects and Issues]
### Select Topic Area

Bug

### Body

All has been very well until today.
I can't understand or bypass the following error on various repos.
Here is one example:
![image](https://user-images.githubusercontent.com/13039271/210431490-9dd0a5f4-d7d8-4639-bc8c-378472732ac3.png)

**Comment by CodesdaLu** [comment]
Your github is not found, it was probably deleted your repository. 
If you use GitHub Desktop see if you have a folder on your pc.
If your repository private exist in your Github not exist this problem

**Reply by keesh0** [reply]
Thanks.
Visual Studio 2019 works fine.
Something is really sick with my IntelliJ GitHub bindings.
I will investigate it.

**Reply by CodesdaLu** [reply]
IntelliJ is very heavy, it may be harming the machine. And causing this problem.

You can look for another IDE to build your codes

Hope this helps
